plugins/base/src/main/kotlin/transformers/documentables/DefaultDocumentablesToPageTranslator.kt

```diff
@@ -1,14 +1,13 @@
-package org.jetbrains.dokka.transformers.documentation
+package org.jetbrains.dokka.base.transformers.documentables
```

Comment:
IMHO it would be best to move translators into separate `translators` package. They don't share common code nor abstractions